### ISSUE TITLE
Scientific (real-world) validity of UXsim simulations

### ISSUE THREAD

**Issue by EwoutH** [toruseo/UXsim#95]
While UXsim is pretty wel rooted in theory in [its paper](https://arxiv.org/abs/2309.17114), I think the next step is to show it simulations are valid and consistent with the real-world. There are multiple approaches to this, but I think one of the ways to go is to compare UXsim with other traffic simulation models, and show that results are consistent and roughly similar.

It would also be interesting to show at which spatial and temporal resolutions UXsim starts to break down (how rough is too rough) and at what hyperparameters UXsim starts to be inaccurate (like platooning group size).

This could be shown in a notebook or two, and tested with each release. The results could be included in the docs and the scientific paper.

**Comment by toruseo** [comment]
Good point, but this will be very tough job. I have some plans, but I cannot address them right now.

Since the underlying traffic flow and route choice models are standard in the literature, I think we can say that it is acceptable for academic research in many situations. But as you pointed out, "how much value is acceptable for the platooning size `deltan`" is an important open question.

In fact, I have partially validated in the Chicago data uploaded today:
https://toruseo.jp/UXsim/docs/notebooks/demo_notebook_08en_chicago.html#Validation
I have compared UXsim's result and a reference solution of static user equilibrium assignment. Because the assignment principles are different we cannot say for sure, but I think we can say the results are a kind of plausible.

**Comment by EwoutH** [comment]
Thanks! If you have any more ideas, just post them here, and we will see who will get to them first! :)

**Comment by toruseo** [comment]
@EwoutH 
BTW, is it possible to share your model by adding it to [demos_and_examples](https://github.com/toruseo/UXsim/tree/main/demos_and_examples) in this repo?

The current examples are all written by me and reflect my coding style. It would be very informative for me and other users if there were examples written by other people (especially those who are familiar with modern sophisticated styles).

**Comment by EwoutH** [comment]
Certainly! I'm still developing and calibrating it and so, but when it's (close to) finished I will definitely share it.

It will be an agent-based model (likely using [Mesa](https://github.com/projectmesa/mesa)) using UXsim, so I expect it will offer a very nice example what's possible with such a hybrid model.
